scss/xy-grid/_layout.scss

```diff
@@ -28,6 +28,6 @@
   $size: percentage(1/$n);
 
   & > #{$selector} {
-    @include xy-cell($size, $gutter-output, $gutters, $gutter-type, $gutter-position, $breakpoint, $vertical);
+    @include xy-cell($size, $gutter-output: $gutter-output, $gutters: $gutters, $gutter-type: $gutter-type, $gutter-position: $gutter-position, $breakpoint: $breakpoint, $vertical: $vertical);
```

Comment:
I don't understand. The `xy-cell` mixin now is:
```scss
@mixin xy-cell(
  $size: full,
  $gutter-output: true,
  $gutters: $grid-margin-gutters,
  $gutter-type: margin,
  $gutter-position: right left,
  $breakpoint: null,
  $vertical: false
) {
```
Why using named parameters here ? Isn't the order right ?

Reply:
Documentation for `xy-grid-layout` is out-of-date. `$breakpoint` description is missing.

Reply:
The named params were set by kball, I will choose the short form here.

Reply:
No I mean: 
```diff
-  @include xy-cell($size, $gutter-output, $gutters, $gutter-type, $gutter-position, $breakpoint, $vertical); 
```
is the same as
```diff
+  @include xy-cell($size, $gutter-output: $gutter-output, $gutters: $gutters, $gutter-type: $gutter-type, $gutter-position: $gutter-position, $breakpoint: $breakpoint, $vertical: $vertical); 
```

Why changing ?

Reply:
> Named params should be more clear here.

Ah, same for here ?

Reply:
`$gutters` and `$breakpoint` were added so the order is not right anymory

Reply:
I don't get it.

Take a look at the call:
```diff
-  @include xy-cell($size, $gutter-output, $gutters, $gutter-type, $gutter-position, $breakpoint, $vertical); 
```
...and the mixin: 
```scss
@mixin xy-cell(
  $size: full,
  $gutter-output: true,
  $gutters: $grid-margin-gutters,
  $gutter-type: margin,
  $gutter-position: right left,
  $breakpoint: null,
  $vertical: false
) {
```

The order is the same.

Reply:
`xy-grid-layout($n: $i, $selector: '.cell', $gutter-output: false, $gutter-type: padding, $breakpoint: $-zf-size);`

It is about `xy-grid-layout` not the cell.

Reply:
See the current changes.

Reply:
And https://github.com/zurb/foundation-sites/pull/11145/files#diff-31c7029afafde5d1c8f61e2705198579R206 was the real bug.

Reply:
I can revert this single line but so far the line was not consistent. The bug was in another line (the one with the breakpoint).

Reply:
Yeah I seen that later.

Reply:
Also the order is not correct. Unnamed arguments could be problematic as there is `gutters` in between. https://github.com/zurb/foundation-sites/blob/develop/scss/xy-grid/_layout.scss#L18

Reply:
> as there is gutters in between

You are passing `$gutters`. You pass all parameters with the same name in the same order to `xy-cell`.

Reply:
You are still referring to the wrong code which is not in the changeset anymore afaik ;-) The change there was reverted.

AI am speaking about https://github.com/zurb/foundation-sites/pull/11145/files#diff-31c7029afafde5d1c8f61e2705198579R191

Reply:
Please check the current changeset here https://github.com/zurb/foundation-sites/pull/11145/files ;-)

I think there is some miscommunication.

Reply:
> Please check the current changeset here https://github.com/zurb/foundation-sites/pull/11145/files ;-)

Yes I seen that. But you seemed to still argue that these are not equivalent.

Reply:
Then it was a fault on my side, sorry.